tests/review-pr-phase3-ci.test.sh

```
@@ -655,6 +655,145 @@ else
+    document("AMBIGUOUS", "code_bug", '"README.md:1"', '"x"'),
+    # a fence that is not the LAST thing in the document
+    document("CLASSIFIED", "code_bug", '"README.md:1"', '"x"') + "trailing content\n",
+    # documents with no fence at all
+    "no fenced document here\n",
+    "```yaml\nstatus: CLASSIFIED\n",
+]
+for index, case in enumerate(cases, start=1):
+    (out / f"case-{index:02d}.md").write_text(case, encoding="utf-8")
+print(len(cases))
+PY
+CLASSIFY_DIFF_COUNT="$(ls "$CLASSIFY_DIFF_DIR" | grep -c '^case-')"
```

Shellcheck (0.11.0)

[warning] Don't use ls | grep. Use a glob or a for loop with a condition to allow non-alphanumeric filenames.

(SC2010)